website/pages/en/billing.mdx

```diff
@@ -2,10 +2,32 @@
 title: Billing
 ---
 
-## Billing on Arbitrum
-
 > Invoices are generated on a weekly basis.
 
+There are two options for paying for your query fees:
+
+- [Paying with fiat currency through Banxa](#billing-on-banxa)
+- [Paying with GRT on Arbitrum](#billing-on-arbitrum)
+
+## Billing on Banxa
```

Comment:
Is it "on Banxa" or "with Banxa"

Reply:
I'm personally a fan of "with Banxa" but I'd love to see what @brandonleekramer & @dnfodjo think

Reply:
I like "with Banxa"